### DISCUSSION TITLE
ScriptReference/TextureImporter.GetPlatformTextureSettings

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#621] [Notes]
# ScriptReference/TextureImporter.GetPlatformTextureSettings

https://docs.unity3d.com/ScriptReference/TextureImporter.GetPlatformTextureSettings.html

**Comment by ggevrin** [comment]
platform is a string and doc is incomplete: use "PS5" for PS5, "GameCoreXboxSeries" for XBOX Series. It looks like the values in BuildTarget enum (take it with a pinch of salt, just guessing).
